images/linux/scripts/installers/azpowershell.sh

```diff
@@ -15,6 +15,10 @@ else
     versions=$(jq -r '.azureModules[] | select(.name | contains("az")) | .versions[]' $toolset)
 fi
 
+# Update PowerShellGet to eliminate download errors
+pwsh -Command "Install-Module -Name PowerShellGet -Force"
+pwsh -Command "Update-Module -Name PowerShellGet -Force"
```

Comment:
Why do we need right after the install Update-Module?

Reply:
true, Install-module updates it to the latest version itself, I'll remove unnecessary Update-Module